active_storage_base64.gemspec

```diff
@@ -15,9 +15,11 @@ Gem::Specification.new do |s|
   s.required_ruby_version = '>= 2.7.0'
 
   # Dependencies
-  s.add_dependency 'rails', '>= 7.0'
+  s.add_dependency 'activestorage', '>= 7.0'
+  s.add_dependency 'activesupport', '>= 7.0'
 
   # Development dependencies
+  s.add_development_dependency 'rails', '>= 7.0'
```

Comment:
if we want to replace the dependency on `rails` for `activestorage` and `activesupport` why do we need this line?

Reply:
I left it for a dummy rails application

Reply:
So? I understand that tests is now the issue, but nobody told me that this PR is ok

Reply:
there are some failing tests but don't seem to be related to your PR. I'll check them out